lib/views/game.html.erb

```diff
@@ -9,46 +9,54 @@
   <h1>Hello, <%= web.name %>!</h1>
 
   <div class="invitation">
-    <p>You have <%= web.game.turns_left %> attempts and <%= web.game.hints_left %> hints. Good luck</p>
-
+    <p>
+      You have <%= web.game.turns_left %>
+      attempts and <%= web.game.hints_left %> hints. Good luck
+    </p>
     <% web.hints.length.times do |index| %>
       <p class="hint">
         <%= web.hints[index] %>
       </p>
     <% end %>
-
   </div>
 
   <% if web.status != :first_step && web.status != :hint %>
```

Comment:
better to use templates for those cases
check game state out of the view and render the template that needed